.github/workflows/playground_examples_ci_reusable.yml

```diff
@@ -116,18 +117,32 @@ jobs:
       - name: Set Docker Tag If Github Tag was trigger
         run: echo "DOCKERTAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
         if: startsWith(github.ref, 'refs/tags/')
-      - name: Set docker image
-        run: echo "IMAGE_TAG=apache/beam_playground-backend-$SDK:$DOCKERTAG" >> $GITHUB_ENV
 
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+            cache-read-only: false
+
+      - name: Build Python SDK runner base image
+        if: env.SDK == 'python'
+        run: |
+             set -uex
+             ./gradlew -i :sdks:python:container:py37:docker -Pdocker-tag="$DOCKERTAG"
+             echo "SDK_TAG=${DOCKERTAG}" >> $GITHUB_ENV
+      
+      # TODO build from the current ./sdks/... for Java and Go
       - name: Build SDK Backend Docker image
         run: |
              set -uex
-             opts="-Pdocker-tag=$DOCKERTAG"
-             if [ "$SDK" == "java" ]; then
-              opts="$opts -Pbase-image=apache/beam_java8_sdk:$BEAM_VERSION"
+             opts=" -Pdocker-tag=$DOCKERTAG"
+             if [ "$SDK" == "python" ]; then
+                opts="$opts -Psdk-tag=$SDK_TAG"
```

Comment:
This is probably cleaner/more extensible as `if [ "$SDK_TAG" != "" ]; then` - it took me a bit to piece together why this was gated on python and requires knowledge of the previous step